src/Benchmarks/Program.cs

```diff
@@ -133,6 +140,21 @@ private static int GetThreadCount(IConfigurationRoot config)
             var threadCountValue = config["threadCount"];
             return threadCountValue == null ? -1 : int.Parse(threadCountValue);
         }
+
+        private static IConnectionFilter GetConnectionFilter(IConfigurationRoot config, IConnectionFilter prevFilter)
+        {
+            var connectionFilterValue = config["connectionFilter"];
+            if (string.IsNullOrEmpty(connectionFilterValue))
+            {
+                return null;
+            }
+            else
+            {
+                var connectionFilterType = Type.GetType(connectionFilterValue, throwOnError: true);
```

Comment:
While I agree we may never actually use this to test another connection filter, it seems like a better design than adding a very specific parameter (and column in SQL) like "passthroughConnectionFilter".